src/docs/sdk/features.mdx

```diff
@@ -268,6 +268,36 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+
+### Scrubbing Sensitive Data
+
+HTTP Client Integrations record the URLs of HTTP requests. URLs can contain two categories of sensitive information:
+* Privacy related data like passwords, private keys, etc.
+* Personally identifiable information (PII) like email addresses, addresses, names, social security numbers, etc.
+
+#### Privacy Related Data
+
+Privacy related data from the `url` in breadcumbs and `$url` in span descriptions must always be scrubbed.
```

Comment:
If you form all sentences in the active voice, you have to specify the subject. Now I could ask myself, who is doing the scrubbing? The server, the SDK? Changing passive to active voice applies to the whole section.

```suggestion
The SDKs must scrub privacy-related data from the `url` in breadcumbs and `$url` in span descriptions.
```